lib/commands/element.js

```diff
@@ -246,6 +252,75 @@ commands.clear = async function (el) {
   await retry(5, this.proxyCommand.bind(this), `/element/${el}/clear`, 'POST');
 };
 
+commands.getContentSize = async function (el) {
+  if (this.isWebContext()) {
+    throw new errors.NotYetImplementedError('Support for getContentSize for webcontext is not yet implemented. Please contact an Appium dev');
+  }
+
+  const type = await this.getAttribute('type', el);
+
+  if (type !== "XCUIElementTypeTable" &&
+      type !== "XCUIElementTypeCollectionView") {
+    throw new Error(`Can't get content size for type '${type}', only for ` +
+                    `tables and collection views`);
+  }
+  let locator = "*";
+  if (type === "XCUIElementTypeTable") {
+    // only find table cells, not just any children
+    locator = "XCUIElementTypeCell";
+  }
+
+  let contentHeight = 0;
+  let children = await this.findElOrEls(`class chain`, locator, true, el);
```

Comment:
is it `class chain` or `-ios class chain`?

Reply:
at this point it can be either. `-ios class chain` is rewritten in a downstream method to `class chain` for WDA